### PR TITLE
Travis warnings

### PR DESCRIPTION
**Pull request by erasche**
- My editor fixes whitespace, so y'all get that for free.
- fixed all of the ordering issues
- STILL missing tests on some things ( e.g. #37 )
- lots of missing citations. I tend to add in a `<citations></citations>` block just to get the warning to go away.

notest (at jenkins)

### CODE REVIEW

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/153/
Test (probably) passed

File: data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_databases.xml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+    <requirements>
+        <requirement type="package" version="4.0">snpEff</requirement>
+    </requirements>
+    <stdio>
+      <exit_code range=":-1"  level="fatal"   description="Error: Cannot open file" />
+      <exit_code range="1:"  level="fatal"   description="Error" />
+    </stdio>
+    <command interpreter="python">
         data_manager_snpEff_databases.py --jar_path \$SNPEFF_JAR_PATH/snpEff.jar "$out_file"
         </command>
```
Comment (nsoranzo) [inline data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_databases.xml]: The closing `</command>` is not aligned correctly.

Comment (peterjc) [inline data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_databases.xml]: Yes, inconsistent indentation. This reminds me I've been meaning to go over my XML files to remove any tabs and use consistent 4 space indentation as per https://github.com/galaxy-iuc/standards/blob/master/docs/best_practices/tool_xml.rst

File: tools/gatk2/variant_combine.xml [hunk 2]
```diff
@@ -30,10 +30,10 @@
    --genotypemergeoption "${genotype_merge_option}"
    --rod_priority_list "${ ','.join( $priority_order ) }"
    '
-   
+
     #include source=$standard_gatk_options#
-    
-    
+
+
```
Comment (nsoranzo) [inline tools/gatk2/variant_combine.xml]: One empty line can be removed.

File: tools/gatk2/variant_combine.xml [hunk 3]
```diff
@@ -79,34 +79,34 @@
...
       <option value="UNSORTED" />
       <option value="REQUIRE_UNIQUE" />
     </param>
-    
+
     <expand macro="gatk_param_type_conditional" />
-    
-    
+
+
```
Comment (nsoranzo) [inline tools/gatk2/variant_combine.xml]: One empty line can be removed.

File: tools/gatk2/variant_combine.xml [hunk 4]
```diff
@@ -79,34 +79,34 @@
...
         <param name="set_key" type="text" value="" label="Key, by default set, in the INFO key=value tag emitted describing which set the combined VCF record came from." help="-setKey,--setKey &amp;lt;setKey&amp;gt;"/>
         <param name="assume_identical_samples" checked="false" type="boolean" truevalue="--assumeIdenticalSamples" falsevalue="" label="If true, assume input VCFs have identical sample sets and disjoint calls so that one can simply perform a merge sort to combine the VCFs into one, drastically reducing the runtime." help="-assumeIdenticalSamples,--assumeIdenticalSamples" />
         <param name="minimum_n" type="integer" value="1" label="Combine variants and output site only if variant is present in at least N input files." help="-minN,--minimumN &amp;lt;minimumN&amp;gt;"/>
-        
+
     </expand>
-    
-    
+
```
Comment (nsoranzo) [inline tools/gatk2/variant_combine.xml]: One empty line can be removed.

File: tools/gatk2/variant_eval.xml [hunk 5]
```diff
@@ -25,24 +25,24 @@
...
             -p '--known_names "${dbsnp_rod_bind_type.dbsnp_rod_name}"'
         #end if
     #end if
-   
+
     #include source=$standard_gatk_options#
-    
-    
+
+
```
Comment (nsoranzo) [inline tools/gatk2/variant_eval.xml]: One empty line can be removed.

File: tools/gatk2/variant_eval.xml [hunk 6]
```diff
@@ -140,10 +140,10 @@
         <param name="dbsnp_known_names" type="boolean" label="Use dbSNP ROD file as known_names" help="-knownName,--known_names &amp;lt;known_names&amp;gt;" />
       </when>
     </conditional>
-    
+
     <expand macro="gatk_param_type_conditional" />
-    
-    
+
+
```
Comment (nsoranzo) [inline tools/gatk2/variant_eval.xml]: One empty line can be removed.

File: tools/gatk2/variant_eval.xml [hunk 7]
```diff
@@ -201,17 +201,17 @@
...
         <param name="ancestral_alignments" type="data" format="fasta" optional="True" label="Fasta file with ancestral alleles" help="-aa,--ancestralAlignments &amp;lt;ancestralAlignments&amp;gt;" />
         <param name="known_cnvs" type="data" format="bed,gatk_interval,picard_interval_list" optional="True" label="File containing tribble-readable features describing a known list of copy number variants" help="-knownCNVs,--knownCNVs &amp;lt;knownCNVs&amp;gt;" />
         <param name="strat_intervals" type="data" format="bed,gatk_interval,picard_interval_list" optional="True" label="File containing tribble-readable features for the IntervalStratificiation" help="-stratIntervals,--stratIntervals &amp;lt;stratIntervals&amp;gt;" />
-        
+
     </expand>
-    
-    
+
+
```
Comment (nsoranzo) [inline tools/gatk2/variant_eval.xml]: One empty line can be removed.

File: tools/rglasso/rglasso_cox.xml [hunk 8]
```diff
@@ -621,13 +407,13 @@ do_lasso = function(x=NA,y=NA,do_standard=T,debugOn=T,defaultFam="gaussian",optL
     sink()
     return(NA)
   }
-  
-  
+
+
```
Comment (nsoranzo) [inline tools/rglasso/rglasso_cox.xml]: One empty line can be removed.

**Comment by erasche** [comment]
@nsoranzo will get around to empty line issues shortly.

File: data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_databases.xml (hunk 1 above)
Comment (erasche) [inline data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_databases.xml]: Hmmm, it does look like I've introduced that problem. Will resolve presently.

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/156/
Test (probably) passed

**Comment by erasche** [comment]
Anyone? Comments or is this OK to merge?

**Comment by peterjc** [comment]
I've eyeballed this and it looks fine.

Tip: hide whitespace differences in github with ``?w=1`` on the end of the URL https://github.com/blog/967-github-secrets

**Comment by erasche** [comment]
@peterjc neat, didn't know about the whitespace parameter. Thanks!

**Pull request merged**